k8s/charts/jd-bureau/templates/deployment.yaml

```
@@ -34,7 +34,7 @@ spec:
               mountPath: /app/server/config/environment/secret.js
               subPath: secret.js
             - name: azurekvjd
-              mountPath: /mnt/secrets/{{ .Values.keyvault.name }}
+              mountPath: /mnt/secrets/jurordigital
```

should be the vault name, is there a reason you shortened it?
```suggestion
              mountPath: /mnt/secrets/juror-digital-vault
```

---

i was trying to avoid "-" in the name. I will modify accordingly.